content/docs/command-reference/push.md

```diff
@@ -155,37 +155,51 @@ a [pipeline](/doc/command-reference/pipeline) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+$ dvc dag
+         +---------+
+         | prepare |
```

Comment:
Idk if we want to do this change or just something else completely e.g. `cat dvc.yaml` but seems out of scope for this PR given the title?

Reply:
This is actually based on example get started repo. I excluded the `data/data.xml.dvc`, as it was taking to much space. This is similar to [dag example](https://dvc.org/doc/command-reference/dag#examples)

Reply:
What do you mean this is based on Get Started? No, look at the stage names, Posts.xml.dvc, Posts.tsv.dvc, etc.

Anyway... How is this related to removing "Dvcfile" again? The cat dvc.yaml idea wasn't good, now it's even longer. Way too long. Let's just not change this for now and focus on the original PR scope?

Reply:
Yeah just rever this pleas.e It's being taken care of in #1584

Reply:
P/a I see it's related because this is where `Dvcfile` gets removed. But yeah, lets just revert thia part since it's being fixed elsewhere already. This way we can merge the rest of this work and move on 🙂